### PR TITLE
Navigation-to-corresponding-album--FileInfo

### PR DESCRIPTION
**Pull request by jubitjohn**
## Description

Nativation to corresponding album when the  user clicks on the album cards on the bottom of file info sidebar

### CODE REVIEW

**Comment by abhinavkgrd** [comment]
@jubitjohn, can you confirm the chip colour and padding are correct as per  
figma https://www.figma.com/file/SYtMyLBs5SAOkTbfMMzhqt/ente-Visual-Design?type=design&node-id=8119-59512&t=nQxLsSFPN6BGdwSW-4

File: apps/photos/src/components/Chip.tsx [hunk 1]
```diff
@@ -1,10 +1,6 @@
-import { Box, styled } from '@mui/material';
+import { Button, styled } from '@mui/material';
 import { CSSProperties } from '@mui/material/styles/createTypography';
 
-export const Chip = styled(Box)(({ theme }) => ({
+export const Chip = styled(Button)(({ theme }) => ({
```
Comment (abhinavkgrd) [inline apps/photos/src/components/Chip.tsx:4]: @jubitjohn, can you confirm the chip colour and padding are correct as per  
figma https://www.figma.com/file/SYtMyLBs5SAOkTbfMMzhqt/ente-Visual-Design?type=design&node-id=8119-59512&t=nQxLsSFPN6BGdwSW-4

File: apps/photos/src/components/PhotoViewer/FileInfo/index.tsx [hunk 2]
```diff
@@ -264,13 +268,21 @@ export function FileInfo({
                                     collectionNameMap.has(collectionID)
                                 )
                                 ?.map((collectionID) => (
-                                    <Chip key={collectionID}>
+                                    <Chip
+                                        key={collectionID}
+                                        onClick={() => {
```
Comment (abhinavkgrd) [inline apps/photos/src/components/PhotoViewer/FileInfo/index.tsx:273]: lets add a helper function `onCollectionChipClick`

File: apps/photos/src/pages/gallery/index.tsx [hunk 3]
```diff
@@ -344,7 +344,7 @@ export default function Gallery() {
         }
         const href = `/gallery${collectionURL}`;
         router.push(href, undefined, { shallow: true });
-    }, [activeCollection]);
+    }, []);
```
Comment (abhinavkgrd) [inline apps/photos/src/pages/gallery/index.tsx:347]: why this ?

Comment (jubitjohn) [reply apps/photos/src/pages/gallery/index.tsx:347]: Error: Cancel rendering route: for this Error, Tried removing activeCollection, error was handled , It doesn't seem have affected any other navigations also

Comment (abhinavkgrd) [reply apps/photos/src/pages/gallery/index.tsx:347]: But this `useEffect` won't work without the `activeCollection` dependency no? We need to update the navbar URL when the `collectionID` changes. Can you check if there is an option to check if router render is in progress and await on that if possible and then update the route

Comment (jubitjohn) [reply apps/photos/src/pages/gallery/index.tsx:347]: setTimeout(() => {
    const href = `/gallery${collectionURL}`;
    router.push(href, undefined, { shallow: true });
  }, 0); Will setting such a timeout work?

Comment (abhinavkgrd) [reply apps/photos/src/pages/gallery/index.tsx:347]: How will this help. How have you seen some issues where people have suggested this?

Comment (jubitjohn) [reply apps/photos/src/pages/gallery/index.tsx:347]: When setting a timeout, the issue seems to have resolved, Not sure of it's side effects

File: apps/photos/src/components/Chip.tsx (hunk 1 above)
Comment (jubitjohn) [reply apps/photos/src/components/Chip.tsx:4]: Okay, I have checked and made the corrections

File: apps/photos/src/components/PhotoViewer/FileInfo/index.tsx (hunk 2 above)
Comment (jubitjohn) [reply apps/photos/src/components/PhotoViewer/FileInfo/index.tsx:273]: I have added a function

**Review by abhinavkgrd — APPROVED** [review]

**Pull request merged**